static/app/components/dateTime.tsx

```diff
@@ -57,8 +57,13 @@ class DateTime extends Component<Props> {
       return 'MM/DD/YYYY';
     }
 
-    // Oct 26, 2017 11:30
     if (clock24Hours) {
+      if (seconds) {
+        // Oct 26, 2017 11:30:30
+        return 'MMM D, YYYY HH:mm:ss';
+      }
+
+      // Oct 26, 2017 11:30
       return 'MMM D, YYYY HH:mm';
```

Comment:
Why only for the 24hr clock?

Reply:
the regular clock already does it

Reply:
ic

Reply:
https://github.com/getsentry/sentry/blob/master/tests/js/spec/components/dateTime.spec.jsx#L20